### PR TITLE
add-ci-to-check-eslint-rules

### PR DESCRIPTION
**Pull request by kilianpichard**

### CODE REVIEW

**Review by remirubis — DISMISSED** [review]
Thank you! Since the time I had to take care of the CI :) I'm not put PR - Review OK because this PR is on WIP status but if it's okay you can merge it.

**Comment by kilianpichard** [comment]
@remirubis i removed some rules that were useless to me (we can of course discuss about it ! )

File: src/models/projectModel.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ const mongoose = require('mongoose');
 
 const { Schema } = mongoose;
 
-const Team = mongoose.models.Team;
-const Task = mongoose.models.Task;
+const { Team } = mongoose.models;
+const { Task } = mongoose.models;
```
Comment (remirubis) [inline src/models/projectModel.js]: Can you merge these lines please, thanks for the refacto!

```suggestion
const { Team, Task } = mongoose.models;
```

**Review by remirubis — DISMISSED** [review]
One change, just commit quickly with Github and it's okay for me !

**Review by remirubis — APPROVED** [review]
LGTM! 🚀

**Pull request merged**